docs/hotplug.md

```diff
@@ -0,0 +1,131 @@
+# Hotplug Volumes
+
+KubeVirt now supports hotplugging volumes into a running Virtual Machine Instance (VMI). The volume must be either a block volume or contain a disk image file just like any other regular volume.
+
+## Enable feature gate
+
+In order to enable the HotplugVolumes feature gate, you must add the HotplugVolumes to the list of enabled featureGates.
+
+```yaml
+spec:
+  configuration:
+    developerConfiguration:
+      featureGates:
+      - HotplugVolumes
+```
+
+Once the feature gate is enabled, you will be able to hotplug volumes into a running VMI.
+
+## Virtctl support
+
+In order to hotplug a volume, you must first prepare a volume. This can be done by using a DataVolume (DV). In the example we will use a blank DV in order to add some extra storage to a running VMI
+
+```yaml
+apiVersion: cdi.kubevirt.io/v1beta1
+kind: DataVolume
+metadata:
+  name: example-volume-hotplug
+spec:
+  source:
+    blank: {}
+  pvc:
+    accessModes:
+      - ReadWriteOnce
+    resources:
+      requests:
+        storage: 5Gi
+```
+In this example we are using ReadWriteOnce accessMode, and the default FileSystem volume mode. Volume hotplugging supports all combinations of block volume mode and ReadWriteMany/ReadWriteOnce/ReadOnlyMany accessModes, if your storage supports the combination.
+
+### Addvolume
+
+Now lets assume we have started a VMI like the [Fedora VMI in examples](examples/vmi-fedora.yaml) and the name of the VMI is 'vmi-fedora' we can add the above blank volume to this running VMI by using the 'addvolume' command  available with virtctl
+
+```bash
+$ virtctl addvolume vmi-fedora --volume-name=example-volume-hotplug
+```
+
+This will hotplug the volume into the running VMI, and set the serial of the disk to the volume name. In this example it is set to example-hotplug-volume.
+
+#### Serial
```

Comment:
Can we use the term "serial number" rather than serial (which sounds like serial console)?

Reply:
Sure